### ISSUE TITLE
Screen capture

### ISSUE THREAD

**Issue by mfreiholz** [mfreiholz/ts3video-public#2]
Add the possibility to share desktop with other participants.

Instead of showing the screen inside the basic webcam video frame, it should show a notification or some kind of icon beside the username, which will open a new window with the shared screen.

**Comment by Jonas-Degn** [comment]
Sharing desktop in addition to or in replacement of webcam would be absolutely amazing!

**Comment by Zalatis** [comment]
For me the 0.6 version work on windows server 2012

**Comment by Vaasref** [comment]
You can already use Screen Capture Recorder.
https://github.com/rdp/screen-capture-recorder-to-video-windows-free

**Comment by Vaasref** [comment]
Yeah it as I said it works, screen-capture-recorder is detected as a webcam so you can use it with TS3Video.

Although, the bitrate is so low you should not hope for readable quality.

I did played some couch coop game with it though, using a Voicemeeter to pipe the sound to TS, a VPN, Synergy and JoyToKey, the delay is playable for some games  but the video quality was so low I often got lost in the screen.

Also screen-capture-recorder is really CPU heavy and impact framerate globally.

Super ghetto setup.

That is why I would like more control over bitrate (and framerate).

**Comment by wilsonzatt** [comment]
Vaasref, can you help me a configurate the screen-capture-record for Live streams? Is necessary to put a stream to url? Or just to press start? THANK YOU!

**Comment by furai** [comment]
I'd love to see proper screen share implemented.
